jetty-integrations/jetty-nosql/src/main/java/org/eclipse/jetty/nosql/mongodb/MongoSessionDataStore.java

```diff
@@ -487,12 +495,12 @@ public void doStore(String id, SessionData data, long lastSaveTime) throws Excep
             sets.put(getContextSubfield(__LASTNODE), data.getLastNode());
             version = ((Number)version).longValue() + 1L;
             ((NoSqlSessionData)data).setVersion(version);
-            update.put("$inc", _version1);
+            //update.put("$inc", _version1);
```

Comment:
We need to update the version stored in the session document.

Reply:
I have some doubt for this as the field `$inc` is read no where and only written here but looks to not be used.